src/elixirServer.ts

```diff
@@ -116,6 +116,23 @@ export class ElixirServer {
         this.sendRequest('DEFL', command, resultCb);
     }
 
+    getDocumentation(document: vscode.TextDocument, position: vscode.Position, callback: Function): void {
+        const wordAtPosition = document.getWordRangeAtPosition(position);
+        const word = document.getText(wordAtPosition);
+        if (word.indexOf('\n') >= 0) {
+            console.error('[vscode-elixir] got whole file as word');
```

Comment:
Okay I see. I thought about how to refactor this so we don't have to check this in all methods the use `document.getWordRangeAtPosition()` but every solution I come up with doesn't keep the code as readable as it is. Maybe this is a case where repetition is the best solution.